wpilibc/src/main/native/include/frc/controller/PIDControllerRunner.h

```diff
@@ -0,0 +1,78 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <functional>
+
+#include <wpi/mutex.h>
+
+#include "frc/Notifier.h"
+#include "frc/controller/PIDController.h"
+#include "frc/smartdashboard/SendableBase.h"
+
+namespace frc {
+namespace experimental {
+
+// TODO: remove when this class is moved into frc namespace
+using frc::Notifier;
```

Comment:
This shouldn't be necessary.  Things in nested namespaces can see things in parent namespaces.